sci/hic.py

```diff
@@ -66,11 +66,12 @@ def load_interaction_data(self, contact_file):
         for line in tqdm(oF.readlines(), desc='Reading %s' % contact_file):
             (chrom1, start1, end1, chrom2,
                 start2, end2, count) = line.strip().split()
-            
+
             if count.split('.')[0].isdigit() and start1.isdigit() and start2.isdigit():
                 i = int(start1) / self.res
                 j = int(start2) / self.res
-
+            else:
+                continue
```

Comment:
Fix 3. if this if test failed, I and j will not be declared, thus causing the later reference of I and j raising Undefined local reference error.